src/utils/is.js

```diff
@@ -74,8 +74,10 @@ export function isFormData(object) {
  */
 export const isAbortSignal = object => {
 	return (
-		typeof object === 'object' &&
-		object[NAME] === 'AbortSignal'
+		typeof object === 'object' && (
+			object[NAME] === 'AbortSignal' ||
+			object[NAME] === 'EventTarget'
```

Comment:
I don't understand this. On Node.js v15.3.0, I see
```js
> new AbortController().signal[Symbol.toStringTag]
'AbortSignal'
```
So it seems like a bug to accept non-AbortSignal EventTarget objects.

Reply:
// @worr

Reply:
Just notice this. yea, it's weird that we check for the name `EventTarget`. a proper test was not even created... still using the polyfill for this...

node v12 goes EOL soon, so we are thinking of dropping support for node below v14.17 so that we can start using AbortController/AbortSignal provided by NodeJS so that we can pass this signal directly to node's own http.request.

Will create an issue for this. (targeted for v4)